### DISCUSSION TITLE
Brew install fabric?

### DISCUSSION THREAD

**Discussion by tsantef** [danielmiessler/Fabric#620] [Ideas]
I saw the note that you're working on the go version so maybe you're already doing this but I think it would be more convenient for Mac users to use brew to install. Thanks for working on this project, I'm eagerly waiting to install it after either the go version or brew install drops.

**Comment by daymeijroos** [comment]
For anybody who is interested, i created my own fabric tap
https://github.com/daymeijroos/homebrew-fabric

**Comment by ksylvan** [comment]
@daymeijroos Great job. Please contribute that to the Homebrew maintainers so it's in the official repo.

I find it most convenient to just do "brew install go" and then use `go install` from the repo directly:

```bash
go install github.com/danielmiessler/fabric@latest
```

This method is also fairly OS-agnostic (I do the same thing on my Win11 and Linux boxes, essentially).

In my case, I also keep a fork of the repo and can do `go install .` in that directory to update my local fabric.

The current pace of development makes it hard to keep the tap updated (since you'd have to update the version number and the shasum a few times a week).

**Reply by bledburn** [reply]
They made an 'official brew installation' formulae for fabric. It's listed as "fabric-ai" in brew. I was making my own brew tap, when i discovered they made this update. It installs as fabric-ai, so there might be an alias or two you want to make...

**Reply by ksylvan** [reply]
> They made an 'official brew installation' formulae for fabric. It's listed as "fabric-ai" in brew. I was making my own brew tap, when i discovered they made this update. It installs as fabric-ai, so there might be an alias or two you want to make...

Yes.

See the official README about that:

https://github.com/danielmiessler/fabric?tab=readme-ov-file#using-package-managers

NOTE: using Homebrew or the Arch Linux package managers makes fabric available as fabric-ai, so add the following alias to your shell startup files to account for this:

```bash
alias fabric='fabric-ai'
```
